mem.go

```diff
@@ -2,12 +2,61 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
-	"github.com/shirou/gopsutil/mem"
+	"github.com/dmmcquay/likexian/host-stat-go"
```

Comment:
You have this import statement incorrect... It shouldn't be nested inside of my user name.

If you did just a regular:

$go get github.com/likexian/host-stat-go

it would put the package in the right place. 

On another note, just a quick glance at that package, I think we could do something else to get hostname (that is what you are using it for).